test/index.test.js

```diff
@@ -1148,7 +1168,8 @@ jobs:
 
             return scm.decorateUrl({
                 scmUri,
-                token: 'mytokenfortesting'
+                token: 'mytokenfortesting',
+                scmContext: 'github.com'
```

Comment:
Ah I see. You need it because the `scm-router` needs it, and they both extend from `scm-base`. How about @minz1027's suggestion here: https://github.com/screwdriver-cd/scm-router/pull/1#discussion_r129989091?
Then we won't need the change to `data-schema` to add `scmContext`, and reduce a lot of extra logic.